app/main.py

```diff
@@ -1 +1,46 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self):
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        print(f"Eating {self.appetite} food points...")
+        was_hungry = self.is_hungry
+        self.is_hungry = False
+        if was_hungry:
+            return self.appetite
+        return 0
```

Comment:
I'm not sure if this is a problem or not, but I think it's a bit ugly
Maybe better don't use extra variable
`        if self.is_hungry:
            print(f"Eating {self.appetite} food points...")
            self.is_hungry = False
            return self.appetite`

Reply:
Should we feed animal if it's not hungry? Nothing said about that in readme